_stories/en/1-covid-19-spread.md

```diff
@@ -0,0 +1,11 @@
+---
+index: 1
```

Comment:
Beetje een terzijde, maar ik vraag me af of we dit soort complete frontmatter settings kunnen uitleggen in een comment binnen dit blok.